src/Doctrine/EventSubscriber/ShopsEventsForTypesenseSubscriber.php

```diff
@@ -5,31 +5,43 @@
 use ACSEO\TypesenseBundle\Manager\CollectionManager;
 use ACSEO\TypesenseBundle\Manager\DocumentManager;
 use AppBundle\Entity\LocalBusiness;
+use AppBundle\Entity\Sylius\Product;
 use AppBundle\Enum\FoodEstablishment;
 use Doctrine\ORM\Events;
 use Doctrine\Common\EventSubscriber;
 use Doctrine\Persistence\Event\LifecycleEventArgs;
```

Comment:
You can use `Doctrine\ORM\Event\LifecycleEventArgs` instead to get rid of PHPStan's error.